correios/client.py

```diff
@@ -99,6 +107,20 @@ def __init__(
 
         self.model_builder = ModelBuilder()
 
+    def _parse_exception(self, exception):
+        message = str(exception)
+
+        if "autenticacao" in message:
+            return AuthenticationError("Authentication error for user {}".format(self.username))
+
+        if message.startswith("O Cartão de Postagem") and message.endswith("Cancelado."):
+            return CanceledPostingCardError("The posting card is canceled")
+
+        if message.startswith('Cartao de Postagem inexistente'):
+            return NonexistentPostingCardError("Nonexistent posting card")
```

Comment:
porque não fazemos algo +/- assim:

```python
ERROR_MESSAGES = {
   re.compile(r"autenticacao"): AuthenticationError,
   re.compile(r"^O Cartão de Postagem.*Cancelado.$"): CanceledPostingCardError,
   # etc...
}

for regex, exc in ERROR_MESSAGES.items():
    if regex.search(message):
        return exc()
```

Reply:
Nice, I'll do the changes.

Reply:
se precisar de ajuda com as regex me avisa...